radicle-httpd/src/api/v1/projects.rs

```diff
@@ -47,8 +48,14 @@ pub fn router(ctx: Context) -> Router {
         .route("/projects/:project/remotes/:peer", get(remote_handler))
         .route("/projects/:project/blob/:sha/*path", get(blob_handler))
         .route("/projects/:project/readme/:sha", get(readme_handler))
-        .route("/projects/:project/issues", get(issues_handler))
-        .route("/projects/:project/issues/:id", get(issue_handler))
+        .route(
+            "/projects/:project/issues",
+            post(issue_create_handler).get(issues_handler),
+        )
```

Comment:
any reason why this is name `issue_create_handler` and not `create_issue_handler`?

Reply:
Hey @xphoniex, no specific reason, I think I wanted to keep starting with the corresponding issue like `project...` `remotes...` etc.

Reply:
nitpick: that's what I was assuming, maybe go with `issue_creation_handler`?

Reply:
yeah sounds good 👍

Reply:
Done in c9834bd